src/catalog/src/local/memory.rs

```diff
@@ -99,7 +117,8 @@ impl CatalogManager for MemoryCatalogManager {
     }
 
     async fn register_system_table(&self, _request: RegisterSystemTableRequest) -> Result<()> {
-        unimplemented!()
+        // TODO(ruihang): support register system table request
+        Ok(())
```

Comment:
return `Unimplemented`?

Reply:
Return error here will fail the start procedure due to ScriptTable cannot be registered. This empty fn simply ignores all those preserved tables.